std_msgs/Header  header
Path[] paths
---
bool success
float64 executed_start_time
float64 executed_path_length
geometry_msgs/Pose goal_pose

---
geometry_msgs/Pose current_pose
int32 current_waypoint_id